extensions.json

```
@@ -379,6 +379,11 @@
       "version": "0.30.3",
       "checkout": "v0.30.3"
     },
+    {
+      "id": "korekontrol.saltstack",
+      "repository": "https://github.com/korekontrol/vscode-saltstack",
+      "version": "0.0.8"
```

It seems fine to always publish from the default branch, so no need to pin a specific version:

```suggestion
      "repository": "https://github.com/korekontrol/vscode-saltstack"
```